FE/src/App.jsx

```diff
@@ -8,6 +8,10 @@ import NewRecipe from './components/NewRecipe'
 import EditRecipe from './components/EditRecipe'
 import Login from './components/Login'
 import Account from './components/Account'
+import UserRecipes from './components/UserRecipes'
```

Comment:
You won't need to import these here, just inside of account. I left a detailed comment on the routes below.